neutron/assetlist.json

```diff
@@ -1242,6 +1242,42 @@
           }
         }
       ]
+    },
+    {
+      "description": "Mars Protocol is a cross-collateralized Money Market Protocol on Neutron and Osmosis.",
```

Comment:
Can we also add an `extended_description` here as well? It has become a requirement for 'verified' status on Osmosis Zone (which I assume is desired due to the modification of the osmosis/assetlist.json)